src/librustc/mir/cache.rs

```diff
@@ -57,7 +57,7 @@ impl Cache {
     }
 }
 
-fn calculate_predecessors(mir: &Mir<'_>) -> IndexVec<BasicBlock, Vec<BasicBlock>> {
+fn calculate_predecessors(mir: &Body<'_>) -> IndexVec<BasicBlock, Vec<BasicBlock>> {
```

Comment:
@oli We might need a separate issue for replacing `mir` used in the names of `mir::Body`s (I don't think we can do too, in this PR, it would bitrot much faster then).

Reply:
makes sense, we can just keep the issue open and make it track multiple points